app/src/main/java/com/kirylshreyter/diamondapp/components/fragments/AboutFragment.java

```diff
@@ -0,0 +1,72 @@
+package com.kirylshreyter.diamondapp.components.fragments;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ProgressBar;
+import android.widget.TextView;
+
+import com.kirylshreyter.diamondapp.R;
+import com.kirylshreyter.diamondapp.common.Constants;
+
+public class AboutFragment extends Fragment {
+    @Nullable
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        return inflater.inflate(R.layout.fragment_about, container, false);
+    }
+
+    @Override
+    public void onActivityCreated(@Nullable Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+        loadFragmentState(savedInstanceState);
+    }
+
+    @Override
+    public void onSaveInstanceState(@NonNull Bundle outState) {
+        super.onSaveInstanceState(outState);
+        saveFragmentState(outState);
+    }
+
+    @Override
+    public void onViewStateRestored(@Nullable Bundle savedInstanceState) {
+        super.onViewStateRestored(savedInstanceState);
+        loadFragmentState(savedInstanceState);
+    }
+
+    private void loadFragmentState(Bundle savedInstanceState) {
+        if (savedInstanceState != null) {
+            TextView text;
+            ProgressBar progressBar;
+            try {
+                text = getActivity().findViewById(R.id.about_activity_text_view);
+                progressBar = getActivity().findViewById(R.id.about_activity_progress_bar);
+            } catch (NullPointerException exception) {
```

Comment:
it's wrong catch nullPointerException. You need to remove it and fix root of issue if nullPointerException happening